client/css/_base.scss

```diff
@@ -4,15 +4,21 @@ html {
   box-sizing: border-box;
 }
 
-*, *:before, *:after {
+*, *:before,
+*:after {
   -webkit-box-sizing: inherit;
   -moz-box-sizing: inherit;
   box-sizing: inherit;
 }
 
 body {
-  padding: 0;
-  #profile { background: $bg-white;}
+  padding: $padding-zero;
+  margin: $margin-zero;
+  font: normal (#{$font-md} / 1.3) $font-stack;
```

Comment:
Question: What do you think about continuing to use the mixin style of `@include font-face(600, $font-lg, normal, $black);`? Maybe we'll need to add a $font-stack parameter to the font-face mixin.

Reply:
Good idea, I was afraid of using that mixin without asking for "permission" first, but I'll add a new param to it.

Reply:
@dantesolis Thanks! Let me know if you need any help. :)